### PR TITLE
fix: Add function validateVariadicArgs

### PR DESCRIPTION
**Pull request by Edmond-J-A**
Signed-off-by: Edmond <edomondja@gmail.com>

### CODE REVIEW

File: src/util/BuiltInFunctions.lua [hunk 1]
```diff
@@ -17,11 +17,21 @@ local posix = require("posix.fnmatch")
...
+function BuiltInFunctions.validateVariadicArgs(expectedLen, args)
+    if #args ~= expectedLen then
+        return error("Expected"..expectedLen.." arguments, but got "..#args)
+    end
+    for i=1,expectedLen do
+        if type(args[i])~="string" then
+            return error("Argument must be a string")
+        end
+    end
+    return nil
```
Comment (rushitote) [inline src/util/BuiltInFunctions.lua:29]: I think this line can be removed since by default any function returns nil.

Comment (Edmond-J-A) [reply src/util/BuiltInFunctions.lua:29]: Got it,fixed

Comment (rushitote) [reply src/util/BuiltInFunctions.lua:29]: @Edmond-J-A Can you check it again, it seems it is still there.

Comment (Edmond-J-A) [reply src/util/BuiltInFunctions.lua]: OK ， I push  it again.

**Review by rushitote — APPROVED** [review]

**Comment by hsluoyz** [comment]
@Edmond-J-A this should be at most a "fix: xxx" instead of "feat: xxx". Fix PR title and commit message.

**Comment by Edmond-J-A** [comment]
> @hsluoyz Changded

**Pull request merged**